### PR TITLE
feat: migrate to SVG icons where possible

### PR DESCRIPTION
**Pull request by jh3y**
- Pull in SVG transformer and set up babel config
- Swap appropriate icons out where applicable
- Any app sweeps appreciated 👍  Think that some of the fill/stroke colors might require changing out.

### CODE REVIEW

File: assets/icons/app/alert.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Icon/24x24/ic-alert</title>
+    <g id="Icon/24x24/ic-alert" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="square">
+        <g id="notification-2" transform="translate(1.000000, 2.000000)" stroke="#FFFFFF" stroke-width="2">
+            <circle id="Oval" cx="11" cy="10" r="4"></circle>
+            <path d="M6,19.8 C2.31557554,17.921696 -0.00400160128,14.1355785 -0.00400160128,10 C-0.00400160128,5.86442152 2.31557554,2.07830395 6,0.2" id="Path"></path>
+            <path d="M16,0.2 C19.6844245,2.07830395 22.0040016,5.86442152 22.0040016,10 C22.0040016,14.1355785 19.6844245,17.921696 16,19.8" id="Path"></path>
+        </g>
+    </g>
+</svg>
```
Comment (ckiss) [inline assets/icons/app/alert.svg:11]: there are some files, like this one, where there is no newline at EOF, would be good to add them

File: assets/icons/index.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  },
+  Settings: Platform.OS === 'ios' ? SettingsIOS : SettingsAndroid
+};
+
+export default {
+  CheckMark,
+  CheckMarkMultiSelect,
+  Logo,
+  Privacy
+};
```
Comment (ckiss) [inline assets/icons/index.ts:82]: not entirely sure why need so many exports. Wouldn't be better to export all of them as default? will ease the import and usage a bit and also when you want to add a new icon you will not have to think about where it fits better.

File: components/atoms/card.tsx [hunk 3]
```diff
@@ -17,11 +12,7 @@ interface CountyBreakdownCardProps {
     v?: number;
     r?: number;
   };
-  icon?: {
-    w: number;
-    h: number;
-    source: ImageRequireSource;
-  };
+  icon?: ReactNode;
```
Comment (ckiss) [inline components/atoms/card.tsx:15]: would be good to have our own specific type for the `icon`

**Review by ckiss — COMMENTED** [review]
good work @jh3y. I left a couple of comments in there, please have a look at let me know your thoughts.

**Review by floridemai — CHANGES_REQUESTED** [review]
The icons come in to replace existing pngs. We should remove those part of this PR imho.
Another thing that we should probably do is to run all svgs through `SVGO` and document that to be part of adding an icon.

**Comment by jh3y** [comment]
@floridemai Updated, wiped cache, removed unused png, added `optimize:svg` run script to run against icons folder 👍

**Comment by ckiss** [comment]
@jh3y @floridemai I tested and works fine. I can't approve though :D LGTM!

**Review by floridemai — APPROVED** [review]
@jh3y Thanks for the follow-ups, looks really good

**Review by ShaunBaker — APPROVED** [review]

**Pull request merged**